### PR TITLE
D785924: move get_instance_count to revision

### PR DESCRIPTION
**Pull request by apollo-crum**

### CODE REVIEW

**Comment by apollo-crum** [comment]
@jsalant22 Can you make sure this doesn't affect any examples, and then move this back to "ready to review"?

File: pyaedt/emit_core/results/revision.py [hunk 1]
```diff
@@ -181,6 +181,25 @@ def is_domain_valid(self, domain):
...
+    @pyaedt_function_handler()
+    def get_instance_count(self, domain):
+        """
+        Return the number of instances in the domain for the current revision.
+
+        Parameters
+        ----------
+        domain :
+            ``InteractionDomain`` object for constraining the analysis parameters.
+
```
Comment (jsalant22) [inline pyaedt/emit_core/results/revision.py:193]:
```suggestion
Returns
--------
count : float
    Number of instances in the domain for the current revision.
```

File: _unittest/test_26_emit.py [hunk 2]
```diff
@@ -692,7 +692,7 @@ def test_availability_1_to_1(self):
         assert domain.interferer_names == ["MD400C"]
         assert domain.interferer_band_names == ["Tx"]
         assert domain.interferer_channel_frequencies == [-1.0]
-        assert domain.instance_count() == 31626
+        assert self.aedtapp.results.revisions[0].get_instance_count(domain) == 31626
         interaction = self.aedtapp.results.revisions[0].run(domain)
```
Comment (MaxJPRey) [inline _unittest/test_26_emit.py:696]:
```suggestion
        revision = self.aedtapp.results.revisions[0]
        assert revision.get_instance_count(domain) == 31626
        interaction = revision.run(domain)
```

File: pyaedt/emit_core/results/revision.py [hunk 3]
```diff
@@ -181,6 +181,28 @@ def is_domain_valid(self, domain):
...
+                domain :
+                    ``InteractionDomain`` object for constraining the analysis parameters.
+        Returns
+        --------
+        count : float
+            Number of instances in the domain for the current revision.
+                Examples
+                ----------
+                >>> domain = aedtapp.interaction_domain()
+                >>> num_instances = aedtapp.results.current_revision.get_instance_count(domain)
```
Comment (MaxJPRey) [inline pyaedt/emit_core/results/revision.py:200]:
```suggestion
        Returns
        --------
        count : float
            Number of instances in the domain for the current revision.
        Examples
        ----------
        >>> domain = aedtapp.interaction_domain()
        >>> num_instances = aedtapp.results.current_revision.get_instance_count(domain)
```

File: pyaedt/emit_core/results/revision.py [hunk 4]
```diff
@@ -181,6 +181,28 @@ def is_domain_valid(self, domain):
...
+        """
+                Return the number of instances in the domain for the current revision.
+
+                Parameters
+                ----------
+                domain :
+                    ``InteractionDomain`` object for constraining the analysis parameters.
+        Returns
+        --------
+        count : float
```
Comment (MaxJPRey) [inline pyaedt/emit_core/results/revision.py:195]: Maybe it could just be an `int` here.
```suggestion
        count : int
```

**Pull request merged**